week6/Homework6-start/Homework6/View/TaskListView.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  TaskListView.swift
+//  
+
+import SwiftUI
+
+struct TaskListView: View {
+  @ObservedObject var taskStore: TaskStore
+    @State var showCompletedTasks: Bool
+    @State private var title = ""
+    
+    var matchingTasks: [Task] {
+        var tasks = taskStore.tasks
```

Comment:
Is there a reason you created a copy of the taskStore.tasks? I returned or filtered it directly and it worked.